src/api/core.js

```diff
@@ -161,23 +162,77 @@ export const makeIntent = (assetAmts, address) => {
   })
 }
 
+/**
+ * @typedef ScriptOutput
+ * @property {string} scriptHash - Uint160
+ * @property {number} operation - string
+ * @property {string} args - array
+ */
+
+/**
+ * Helper method to convert a TokenAmounts object to transfer invocation scripts (ScriptOutput[]).
+ * @param {object} tokenAmts - Token Amounts
+ * @param {number} tokenAmts.SCRIPT_HASH - Amt of [SCRIPT_HASH] token to send. (TODO: better way to write this for docs?)
+ * @param {string} toAddress - The address to send to.
+ * @param {string} fromAddress - The address to send from.
+ * @return {ScriptOutput[]} ScriptOutput
+ */
+export const makeTransfer = (tokenAmts, toAddress, fromAddress) => {

```

Comment:
I don't like the idea of passing `fromAddress` here since it's already part of the config, it seems redundant.  I think I can drop it from here and have this return some other new set of objects instead of `ScriptOutput[]`.  Then `createTransferTx` or `createTransferScript` can be responsible for constructing `args`, which includes the representation of the from address.